### PR TITLE
Fixed #39073 Add STUMPY to Pandas ecosystem docs

### LINKED ISSUE #39073: DOC: Add STUMPY to pandas ecosystem
#### Location of the documentation

[Pandas Ecosystem - Stats & ML](https://pandas.pydata.org/docs/ecosystem.html#statistics-and-machine-learning)

#### Documentation problem

This isn't a problem, just an addition to the current ecosystem listing.

#### Suggested fix for documentation

This is a simple addition to the `ecosystem.rst` file for adding [STUMPY](https://github.com/TDAmeritrade/stumpy) to the existing docs

### PR DESCRIPTION
**Pull request by seanlaw**
- [ ] closes #39073
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

**Comment by seanlaw** [comment]
@jreback As we had briefly discussed, this is a PR to add STUMPY to the `ecosystem.rst` file. Thank you for the suggestion!

**Comment by jreback** [comment]
pre commit checks are complaining

```
@@ -88,9 +88,9 @@ Compose is a machine learning tool for labeling data and prediction engineering.
 `STUMPY <https://github.com/TDAmeritrade/stumpy>`__
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-STUMPY is a powerful and scalable Python library for modern time seris analysis. 
+STUMPY is a powerful and scalable Python library for modern time seris analysis.
 At its core, STUMPY efficiently computes something called a
-`matrix profile <https://stumpy.readthedocs.io/en/latest/Tutorial_The_Matrix_Profile.html>`__, 
+`matrix profile <https://stumpy.readthedocs.io/en/latest/Tutorial_The_Matrix_Profile.html>`__,
 which can be used for a wide variety of time series data mining tasks.
 
 .. _ecosystem.visualization:
```

ping when green.

File: doc/source/ecosystem.rst [hunk 1]
```diff
@@ -85,6 +85,14 @@ Featuretools is a Python library for automated feature engineering built on top
 
 Compose is a machine learning tool for labeling data and prediction engineering. It allows you to structure the labeling process by parameterizing prediction problems and transforming time-driven relational data into target values with cutoff times that can be used for supervised learning.
 
+`STUMPY <https://github.com/TDAmeritrade/stumpy>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+STUMPY is a powerful and scalable Python library for modern time seris analysis.
```
Comment (jreback) [inline doc/source/ecosystem.rst:91]: seris -> series

**Comment by seanlaw** [comment]
@jreback The pre commit is green now

**Comment by jreback** [comment]
thanks @seanlaw

**Pull request merged**